front/app/containers/Admin/projects/project/analysis/FilterItems/index.tsx

```diff
@@ -0,0 +1,143 @@
+
+  const domicileField = customFields?.data.find(
+    (field) => field.attributes.code === 'domicile'
+  );
+
+  const birthyearField = customFields?.data.find(
+    (field) => field.attributes.code === 'birthyear'
+  );
+
+  const { data: genderOptions } = useUserCustomFieldsOptions(genderField?.id);
+  const { data: domicileOptions } = useUserCustomFieldsOptions(
+    domicileField?.id
+  );
+
+  const genderUrlQueryParamKey = `author_custom_${genderField?.id}`;
+  const domicileUrlQueryParamKey = `author_custom_${domicileField?.id}`;
+  const birthyearUrlQueryParamFromKey = `author_custom_${birthyearField?.id}_from`;
+  const birthyearUrlQueryParamToKey = `author_custom_${birthyearField?.id}_to`;
+
+  const translationKeys: Record<string, string> = {
+    search: 'Search',
+    published_at_from: 'Start',
+    published_at_to: 'End',
+    reactions_from: 'Reactions >',
+    reactions_to: 'Reactions <',
+    votes_from: 'Votes >',
+    votes_to: 'Votes <',
+    comments_from: 'Comments >',
+    comments_to: 'Comments <',
+    [genderUrlQueryParamKey]: 'Gender',
+    [domicileUrlQueryParamKey]: 'Domicile',
+    [birthyearUrlQueryParamFromKey]: 'Birthyear >',
+    [birthyearUrlQueryParamToKey]: 'Birthyear <',
+  };
+
+  const filterEntries = Object.entries(filters).filter(
+    ([key]) => key !== 'tag_ids'
+  );
+
+  const filterItemDisplayValue = (
+    key: string,
+    value: string | string[] | undefined | number
+  ) => {
+    switch (key) {
+      case genderUrlQueryParamKey:
+        return (
+          isArray(value) &&
+          value.map((value: string) => (
+            <Box key={value} ml="4px">
+              {localize(
+                genderOptions?.data.find((option) => {
+                  return option.attributes.key === value;
+                })?.attributes.title_multiloc
+              )}
+            </Box>
+          ))
+        );
+      case domicileUrlQueryParamKey:
+        return (
+          isArray(value) &&
+          value.map((value: string) => (
+            <Box key={value} ml="4px">
+              {localize(
+                domicileOptions?.data.find(
+                  (option) => option.attributes.key === value
+                )?.attributes.title_multiloc
+              )}
+            </Box>
+          ))
+        );
+      case birthyearUrlQueryParamFromKey:
+      case birthyearUrlQueryParamToKey:
+        return isString(value) && new Date(value).getFullYear();
+      default:
+        return value;
+    }
+  };
+
+  return (
+    <Box display="flex" flexWrap="wrap" gap="4px">
+      {filterEntries.map(([key, value]) => (
+        <Box
+          key={key}
+          py="4px"
+          px="8px"
+          borderRadius={stylingConsts.borderRadius}
+          borderColor={colors.success}
+          borderWidth="1px"
+          borderStyle="solid"
+          bgColor={colors.white}
+          color={colors.success}
+          display="flex"
+        >
+          {translationKeys[key]}
+          {': '}
```

Comment:
For the `>` and `<` filters, the `:` makes it look weird. Maybe we can drop it altogether?

Reply:
Yes, I agree - it's a bit hacky the way it works now, will try to improve it a bit.